packages/datetime/test/common/dateUtilsTests.tsx

```diff
@@ -265,21 +265,19 @@ describe("dateUtils", () => {
 
     describe("getIsPmFrom24Hour", () => {
         it("returns true, if hour (in 24 range) is PM", () => {
-            const pmHours = [];
-            for (let i = 12; i <= 23; i++) {
-                pmHours.push(i);
-            }
-            for (const hour of pmHours) {
+            expect(DateUtils.getIsPmFrom24Hour(12)).to.equal(true);
+            expect(DateUtils.getIsPmFrom24Hour(23)).to.equal(true);
+
+            for (let hour = 12; hour <= 23; hour++) {
```

Comment:
wait wait the point of my previous comment was to _replace_ all `for` loops with a handful of explicit tests. you added explicit tests but didn't _remove_ the loops.

no need to test 12 and 23 _and then_ test 12 _through_ 23.

(this comment applies to every test in this file. i want to see zero `for` loops when we're done.)

Reply:
does it make sense to explicitly test for all values from 12 to 23? Normally I would just iterate this
**e.g.**
`expect(DateUtils.getIsPmFrom24Hour(12)).to.equal(true);`

`expect(DateUtils.getIsPmFrom24Hour(13)).to.equal(true);`

`expect(DateUtils.getIsPmFrom24Hour(14)).to.equal(true);`

`expect(DateUtils.getIsPmFrom24Hour(15)).to.equal(true);`

`expect(DateUtils.getIsPmFrom24Hour(16)).to.equal(true);`

`expect(DateUtils.getIsPmFrom24Hour(17)).to.equal(true);`

`expect(DateUtils.getIsPmFrom24Hour(18)).to.equal(true);`

`expect(DateUtils.getIsPmFrom24Hour(19)).to.equal(true);`

`expect(DateUtils.getIsPmFrom24Hour(20)).to.equal(true);`

`expect(DateUtils.getIsPmFrom24Hour(21)).to.equal(true);`

`expect(DateUtils.getIsPmFrom24Hour(22)).to.equal(true);`

`expect(DateUtils.getIsPmFrom24Hour(23)).to.equal(true);`